apps/web/lib/team/[slug]/[type]/getServerSideProps.tsx

```diff
@@ -169,11 +207,7 @@ export const getServerSideProps = async (context: GetServerSidePropsContext) =>
           username: orgSlug ?? null,
         },
         title: eventData.title,
-        users: usersInOrgContext.map((user) => ({
-          ...user,
-          metadata: undefined,
-          bookerUrl: getBookerBaseUrlSync(user.profile?.organization?.slug ?? null),
-        })),
+        users,
```

Comment:
This doesn't contain `bookerUrl`. is this backward compatible?

Reply:
We only recently passed users and we only use username and name from users. check out https://github.com/calcom/cal.com/pull/17987 if you need